locales/en-US/notes.properties

```diff
@@ -0,0 +1,13 @@
+welcomeTitle=Welcome!
+welcomeText=This is a simple one-page notepad built in to Firefox that helps you get the most out of the web.
+
+emptyPlaceHolder=Take a note…
+
+openingLogin=Opening Login…
+forgetEmail=Forget this Email
+
+syncNotReady=Sorry, Sync is not ready yet, but we will count your click as a vote to hurry it up!
```

Comment:
Well spotted! I think we should say:

syncNotReady=Sorry, syncing notes isn’t ready. We will counted your click as a vote to hurry it up!

I added a non-breaking space between the last two words to avoid an "orphan" text wrap. Keep it in if possible.